packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -656,7 +662,8 @@ export class TablePageObject<T extends TableRecord> {
         return groupRow.shadowRoot!.querySelector('.group-header-view')!;
     }
 
-    private getRow(rowIndex: number): TableRow {
+    /** @internal */
+    public getRow(rowIndex: number): TableRow {
```

Comment:
Asked this in our team meeting today but there weren't really strong opinions one way or the other. Since there's just 3 internal methods right now I might lean towards keeping it as-is.